src/nvim/digraph.c

```diff
@@ -1442,6 +1442,37 @@ int do_digraph(int c)
   return c;
 }
 
+/*
+ * Find a digraph for "val".  If found return the string to display it.
+ * If not found return NULL.
+ */
+    char_u *get_digraph_for_char(val)
+    int val;
+{
+    int		i;
+    int		use_defaults;
```

Comment:
Loop variables should be declared in the loop.

Reply:
ok will do